docs/faq/frequently_asked_questions.rst

```diff
@@ -0,0 +1,56 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _frequently_asked_questions:
+
+##########################
+Frequently Asked Questions
+##########################
+
+#. I just purchased my motor, how do I get started with Control Center? How can I communicate with my module?
+    * Your module must be configured properly through the Control Center before your module can spin. First, refer to your module’s family page that is found in the sidebar under the Modules section for hardware setup instructions. After your hardware is set up, refer to our :ref:`Control Center Manual <control_center_start_guide>` for instructions on how to set up the Control Center. Then, follow the instructions on configuring your module using Control Center depending on its firmware: :ref:`Speed <speed_module_start_guide>`, :ref:`Servo <servo_module_start_guide>`, or :ref:`Pulsing <pulsing_module_start_guide>`. Please see our documentation under the Tutorials section of the sidebar for additional information. We also have a supplementary application called :ref:`Vertiq Testing Tool <vertiq_testing_tool_guide>` to help you test your module.
+
+#. Why does my module not spin when I send it a PWM signal?
+    * Vertiq modules require initial configuration before being able to spin with PWM commands. You can configure your motor using :ref:`Control Center <control_center_start_guide>`. The most commonly missed configuration when trying to spin with PWM is the :ref:`Motor Direction <throttle_direction>`, which must be set to something other than unconfigured. We recommend setting it to 2D Counter-Clockwise for your initial setup. You should also check what :ref:`Mode <throttle_mode>` is set on your module and that the corresponding :ref:`Maximum <throttle_maximums>`  for that mode is set. If you are still having trouble, refer to the :ref:`PWM and DSHOT tutorial <hobby_fc_tutorial>` or your module's Getting Started documentation: :ref:`Speed <speed_module_start_guide>`, :ref:`Servo <servo_module_start_guide>`, or :ref:`Pulsing <pulsing_module_start_guide>`.
+
+#. I’m trying to use DroneCAN but my motor isn’t spinning.
```

Comment:
"that will arm the module initially" can you link to the arming manual thingy where you say arm?